front/components/app/DatasetPicker.tsx

```diff
@@ -69,16 +62,17 @@ export default function DatasetPicker({
           ) : readOnly ? null : (
             <Menu.Button
               className={classNames(
-                "inline-flex items-center rounded-md py-1 text-sm font-normal",
-                dataset ? "px-0" : "border px-3",
-                readOnly
-                  ? "border-white text-gray-300"
-                  : "border-orange-400 text-gray-700",
+                "inline-flex items-center rounded-md px-3 py-1 text-sm font-normal",
+                dataset ? "border px-1" : "border border-orange-400",
+                readOnly ? "border-white text-gray-300" : "text-gray-700",
                 "focus:outline-none focus:ring-0"
               )}
             >
               {dataset ? (
                 <>
+                  <div className="text-sm font-bold text-action-500">
+                    {dataset}
```

Comment:
How can one edit the dataset from that block? Is that intended that it is not possible anymore? This is OK just want to check it's intentional.

Reply:
It felt quite counterintuitive to click on the dropdown and be redirected to editing the dataset - never seen that in a UI 
So I just added an edit button next to it 
<img width="350" alt="Screenshot 2024-04-29 at 10 22 49" src="https://github.com/dust-tt/dust/assets/1189312/d8f1277d-000d-47f0-89da-eba000d78a54">

Reply:
No a strong reason enough :)

Reply:
Well these tabs are "more engaging" IMHO. And I personally do rely on browser history to move from app to datasets to config etc...

Reply:
> No a strong reason enough :)

I personally never click on the down arrow to open a dropdown and I don't expect clicking in the middle to be link. 
Having a counterintuitive dropdown behavior seems like a quite strong reason to make a small change to me haha, but I can revert if that doesn't seem right. 🥲

Reply:
> Well these tabs are "more engaging" IMHO. And I personally do rely on browser history to move from app to datasets to config etc...

I believe this was meant for the `router.replace` part - reverted, we can decide later.

Reply:
Comments were not in the right place. This change looks good to me :+1: